modules/serverless-kafka-broker-sasl-default-config.adoc

```diff
@@ -46,11 +36,17 @@ $ oc create secret -n knative-eventing generic <secret_name> \
   --from-literal=password="SecretPassword" \
   --from-literal=user="my-sasl-user"
 ----
+.. Use the key names `ca.crt`, `password`, and `sasl.mechanism`. Do not change them.
```

Comment:
`Do not change them.`  To some degree those are part of the API, hence should not be changed by the user. Not sure if we wanna say something more here?

Reply:
I think that's outside the scope of this PR. I didn't _add_ this line here, just moved it to be a sub-point as part of the restructuring, so if there's more info required I'd recommend opening an issue to revisit this.